src/app/modules/project/details/details.component.ts

```diff
@@ -348,6 +350,21 @@ export class DetailsComponent implements OnInit {
     this.displayEmbedButton = this.project.user.id === this.currentUser.id;
   }
 
+  /**
+   * Method to handle the click of the like button
+   * It will either like or unlike the project
+   */
+  public likeClicked() {
```

Comment:
Add return type to the methods. I assume:
```suggestion
  public likeClicked(): void {
```